src/options.ts

```diff
@@ -238,14 +210,14 @@ hljs.registerLanguage('plaintext', plaintext);
       preview.dataset.highlighted = '';
       hljs.highlightElement(preview);
 
-      if (!isValidAo3ValidHtml(template.value)) {
-        templateTextField.helperTextContent =
+      if (!isValidAo3ValidHtml(templateTextField.value)) {
+        templateTextField.errorText =
           'This template appears to contain HTML tags that cannot be used on ' +
```

Comment:
If there's an error, do we prevent saving and/or only save the "sanitized" version? (I would lean towards allowing people to save their input regardless since there may be errors in our validation (cough href), but idk if there's any built-in functionality associated with having an error in a required field that would prevent that)

Reply:
We allow the user to save regardless and just warn them of the issue